atomic_reactor/core.py

```diff
@@ -95,6 +96,8 @@ def _obtain_source_from_path_if_needed(self, local_path, container_path=CONTAINE
         build_json_path = os.path.join(local_path, BUILD_JSON)
         with open(build_json_path, 'r') as fp:
             build_json = json.load(fp)
+
+        validate(build_json, JSON_SCHEMA)
```

Comment:
Ideally we'd show *all* the problems in the input, not just the first, like this: https://github.com/projectatomic/atomic-reactor/blob/master/atomic_reactor/plugins/pre_reactor_config.py#L148